testsuite/standalone/src/test/java/org/wildfly/extras/creaper/commands/elytron/tls/AddKeyStoreOnlineTest.java

```diff
@@ -0,0 +1,120 @@
+package org.wildfly.extras.creaper.commands.elytron.tls;
+
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+import static org.junit.Assert.fail;
+
+import java.io.IOException;
+
+import org.jboss.arquillian.junit.Arquillian;
+import org.junit.After;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.wildfly.extras.creaper.commands.elytron.AbstractElytronOnlineTest;
+import org.wildfly.extras.creaper.commands.security.realms.AddPropertiesAuthentication;
+import org.wildfly.extras.creaper.core.CommandFailedException;
+import org.wildfly.extras.creaper.core.online.operations.Address;
+
+@RunWith(Arquillian.class)
+public class AddKeyStoreOnlineTest extends AbstractElytronOnlineTest{
+
+    private static final String TEST_KEY_STORE_NAME = "CreaperTestKeyStore";
+    private static final String TEST_KEY_STORE_TYPE = "JKS";
+    private static final Address TEST_KEY_STORE_ADDRESS = SUBSYSTEM_ADDRESS.and("key-store", TEST_KEY_STORE_NAME);
+
+    @After
+    public void cleanup() throws Exception {
+        try {
+            ops.removeIfExists(TEST_KEY_STORE_ADDRESS);
+            administration.reloadIfRequired();
+        } finally {
+            client.close();
+        }
+    }
+
+    @Test
+    public void addSimple() throws Exception {
+        AddKeyStore addKeyStore = new AddKeyStore.Builder(TEST_KEY_STORE_NAME, TEST_KEY_STORE_TYPE).build();
+        assertFalse("The key store should not exist", ops.exists(TEST_KEY_STORE_ADDRESS));
+        client.apply(addKeyStore);
+        assertTrue("Key store should be created", ops.exists(TEST_KEY_STORE_ADDRESS));
+    }
+
+    @Test(expected = CommandFailedException.class)
+    public void existing() throws Exception {
```

Comment:
It might be worth considering renaming the test methods to be consistent with the rest of commands. E.g. compare with https://github.com/jboss-security-qe/creaper/blob/elytron/testsuite/standalone/src/test/java/org/wildfly/extras/creaper/commands/elytron/realm/AddAggregateRealmOnlineTest.java#L115